jbehave-core/src/main/java/org/jbehave/core/model/ExamplesTable.java

```diff
@@ -160,7 +160,8 @@
     private static final Map<String, String> EMPTY_MAP = Collections.emptyMap();
     private static final String EMPTY_VALUE = "";
 
-    public static final Pattern INLINED_PROPERTIES_PATTERN = Pattern.compile("\\{(.*?[^\\\\])\\}\\s*(.*)", DOTALL);
+    public static final Pattern INLINED_PROPERTIES_PATTERN = Pattern.compile("\\{(.*?[^\\\\])\\}(?!\\s*[\\},])\\s*(.*)",
```

Comment:
this change should go as a separate commit and should be covered with unit tests